app/models/concerns/exportable.rb

```diff
@@ -157,7 +157,16 @@ def get_field_value_for_serialization(field)
     value = public_send(field)
     if value.is_a?(Array)
       # Use simpler serialization for Array values than the default (`to_s`)
-      value.reject(&:blank?).join(', ')
+      sanitize(value.reject(&:blank?).join(', '))
+    else
+      sanitize(value)
+    end
+  end
+
+  def sanitize(value)
+    if value.is_a?(String) && value =~ /\A\s*[=@+\-\t\r]/
```

Comment:
do you think this makes this more readable? Could go either way
```suggestion
    if value.is_a?(String) && value.starts_with? /\s*[=@+\-\t\r]/
```

Reply:
Good call, I like `starts_with` more too now that you mention it. Changed in new revision. Was getting warnings about ambiguity without parentheses so added parentheses around the regex too.

Reply:
heroic